src/shogun/mathematics/graph/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ set(SHOGUN_CORE_GRAPH_HEADERS
 	${CMAKE_CURRENT_SOURCE_DIR}/Tensor.h
 	${CMAKE_CURRENT_SOURCE_DIR}/Types.h
 	${CMAKE_CURRENT_SOURCE_DIR}/nodes/Add.h
+	${CMAKE_CURRENT_SOURCE_DIR}/nodes/Subtract.h
```

Comment:
@vigsterkr  I know it is meant to be bad practice in cmake, but maybe we should replace some of these with glob?

Reply:
glob is crazy.... but of course we can do it! we are doing it atm, so it's its all fine